docs/INSTALL.md

```diff
@@ -1,47 +1,103 @@
+**Tooling**
 
+This requires `clickable` to be installed locally.
+Installation instructions can be found [here](https://clickable-ut.dev/en/latest/install.html#install).
 
-<!-- ## get go-qml dependencies
+**Dependencies**
+
+The following build dependencies are required:
+* Docker
+* Go
+
+The following go-qml dependencies are required:
 ```
 sudo add-apt-repository ppa:ubuntu-sdk-team/ppa
 sudo apt-get update
 sudo apt-get install qtdeclarative5-dev qtbase5-private-dev qtdeclarative5-private-dev libqt5opengl5-dev qtdeclarative5-qtquick2-plugin
 sudo ln -s /usr/include/x86_64-linux-gnu/qt5/QtCore/5.9.1/QtCore /usr/include/
 ```
 
+To install all go dependencies, use `go mod download`.
 
-## get go lang
+**Build and Install**
 
-```
-# This will give you the latest version of go
-snap install --classic go 
+To run the default set of sub-commands, simply run clickable in the root directory.
+Clickable will attempt to auto detect the build template and other configuration options.
 
-```
--->
+`clickable`
+
+**Run**
+
+`clickable launch`
+
+For a full list of available clickable commands, see [here](https://clickable-ut.dev/en/latest/commands.html).
+
+## Snap
+
+**Tooling**
+
+This requires `snapcraft` to be installed locally.
+Installation instructions can be found [here](https://snapcraft.io/docs/getting-started).
+
+**Dependencies**
+
+TODO: Add more info
+
+**Build and Install**
+
+The Snap template used for the installation can be found
+in the /snap subdirectory.
+
+TODO: Add more info
 
-## set $GOPATH
+**Run**
 
-## get go dependencies
+TODO: Add more info
 
+## Flatpak
+
+**Tooling**
+
+This requires `flatpak` and `flatpak-builder` to be installed locally.
+Installation instructions can be found [here](https://flatpak.org/setup/)
+
+**Dependencies**
+
+The following Flatpak SDKs are required:
 ```
-go get -v -d github.com/sirupsen/logrus
-go get -v -d github.com/godbus/dbus
-go get -v -d github.com/dustin/go-humanize
-go get -v -d github.com/godbus/dbus
-go get -v -d github.com/gosexy/gettext
-go get -v -d github.com/nanu-c/textsecure
-go get -v -d github.com/jmoiron/sqlx
-go get -v -d github.com/mattn/go-sqlite3
-go get -v -d github.com/ttacon/libphonenumber
-go get -v -d github.com/snapcore/snapd/osutil/sys
-go get -v -d github.com/morph027/textsecure
-go get -v -d gopkg.in/yaml.v2
-go get -v -d bitbucket.org/llg/vcard
+flatpak install org.freedesktop.Platform//20.08
+flatpak install org.freedesktop.Sdk//20.08
+flatpak install org.freedesktop.Sdk.Extension.golang//20.08
+flatpak install org.freedesktop.Sdk.Extension.node12//20.08
 ```
+
+**Build and Install**
+
+Installation can be done user-level or system-wide.
+To list installed applications and/or runtimes, use `flatpak list`.
+
+The Flatpak [manifest](https://docs.flatpak.org/en/latest/manifests.html) used for the installation can be found
+in the /flatpak subdirectory.
+
+User-level: 
+
+```flatpak-builder --user --install build org.nanuc.Axolotl.yml```
```

Comment:
In my opinion, we should add in a manner, that it could be copied and pasted and for that, it should be 
```flatpak-builder --user --install build ./flatpak/org.nanuc.Axolotl.yml```

Reply:
Adjusted